### DISCUSSION TITLE
`seaborn.objects` doesn't respect `matplotlibrc` file

### DISCUSSION THREAD

**Discussion by paw-lu** [mwaskom/seaborn#3036] [Q&A] [answered]
Seems that charts created with `seaborn.objects.Plot` don't respect a users `matplotlibrc` file (functional plotting still does though). Is there a way to get plots created from `seaborn.objects` to intake the preferences in `matplotlibrc` file?

<img width="687" alt="Screen Shot 2022-09-19 at 1 51 13 PM" src="https://user-images.githubusercontent.com/30049606/191114153-3f94da0d-f93e-40f9-992f-a42589738dc1.png">

---

Loving the new changes, and really enjoying the `seaborn.objects` interface. Incredible work here.

**Comment by mwaskom** [comment, marked as answer]
This is by design see https://github.com/mwaskom/seaborn/discussions/3005 for more commentary

**Reply by paw-lu** [reply]
Thanks @mwaskom for the pointer!

Seems like similar to what is shown in #3005 ,  we can force the rcParams by adding a `.theme(matplotlib.pyplot.rcParams)` at the end. Modifying my original example:


```python
import matplotlib.pyplot as plt
import numpy as np
import pandas as pd
import seaborn as sns
import seaborn.objects as so


rng = np.random.default_rng(42)
size = 20
df = pd.DataFrame(
    {
        "a": rng.normal(0, 1, size=size),
        "b": rng.normal(1, 2, size=size),
        "c": rng.choice(("a", "b", "c"), size=size),
    }
)
df.pipe(so.Plot, x="a", y="b", color="c").add(so.Dot()).theme(plt.rcParams)  # Change is here!
```

**Reply by mwaskom** [reply]
Oh yes, sorry if that wasn't obvious: that's the best way to accomplish what you want right now.
